src/plone/restapi/services/navroot/get.py

```diff
@@ -28,7 +28,7 @@ def __call__(self, expand=False):
         result["navroot"]["navroot"] = getMultiAdapter(
             (portal_state.navigation_root(), self.request),
             ISerializeToJson,
-        )()
+        )(include_items=False, include_expansion=False)
```

Comment:
@pgrunewald There's a non-zero chance someone has registered a custom serializer which does not accept these keyword args. So we should probably be careful and first try this way, then fall back to calling it with no parameters.

Reply:
Good catch. I added also a test with a custom serializer w/o keyword args to see, if it works.